src/Stratis.Bitcoin.Features.RPC/Controllers/RPCController.cs

```diff
@@ -91,27 +92,28 @@ private RPCResponse SendRPCRequest(RPCRequest request)
         /// </summary>
         /// <returns>A JSON result that varies depending on the RPC method.</returns>
         [Route("callbyname")]
-        [HttpGet]
-        public IActionResult CallByName([FromQuery]string methodName)
+        [HttpPost]
+        public IActionResult CallByName([FromBody]JObject body)
         {
             try
             {
+                var ignoreCase = StringComparison.InvariantCultureIgnoreCase;
```

Comment:
Do we need to have this as a variable?

Reply:
just found it more readable, but I am not too fussed about it honestly.